website/content/sdk/guides/intros/portfolio/index.md

```diff
@@ -1,217 +1,298 @@
+|  7 | 2021-10-29 | STOCK  | K.TO     | Buy    |    7.93 |      190   |     20 |     1526.7   | CAD        | Basic Materials        | Gold                           | Canada        | North America |
+|  6 | 2015-01-02 | ETF    | SPY      | Buy    |  178.28 |        5.6 |      0 |      998.368 | USD        | -                      | -                              | -             | -             |
+|  5 | 2015-01-01 | CRYPTO | BTC-USD  | Buy    | 1000    |          2 |      0 |         2000 | USD        | Crypto                 | Crypto                         | Crypto        | Crypto        |
+|  0 | 2011-01-03 | STOCK  | AMZN     | Buy    |    9.22 |        100 |      0 |          922 | USD        | Consumer Cyclical      | Internet Retail                | United States | North America |
+|  1 | 2011-01-03 | STOCK  | AAPL     | Buy    |   11.74 |        100 |      0 |         1174 | USD        | Technology             | Consumer Electronics           | United States | North America |
+|  2 | 2011-01-03 | STOCK  | MSFT     | Buy    |   28.04 |        100 |      0 |         2804 | USD        | Technology             | Software-Infrastructure        | United States | North America |
+|  3 | 2011-01-03 | STOCK  | TSLA     | Buy    |    1.76 |        100 |      0 |          176 | USD        | Consumer Cyclical      | Auto Manufacturers             | United States | North America |
+|  4 | 2011-01-03 | STOCK  | GOOG     | Buy    |   15.01 |        100 |      0 |         1501 | USD        | Communication Services | Internet Content & Information | United States | North America |
+
+Not every column needs to contain a value, but they must exist despite being empty. The accepted values for `Type` are currently:
+
+- Crypto
+- ETF
+- Stock
+- More Coming Soon!
+
+Below is the contents of the `csv` file that was used in the table above.
+
+```csv
+Date,Ticker,Type,Sector,Industry,Country,Price,Quantity,Fees,Premium,Investment,Side,Currency,Region
+2011-01-03,AMZN,Stock,,,,9.22,100,0,0,922,Buy,USD,
+2011-01-03,AAPL,Stock,,,,11.74,100,0,0,1174,Buy,USD,
+2011-01-03,MSFT,Stock,,,,28.04,100,0,0,2804,Buy,USD,
+2011-01-03,TSLA,Stock,,,,1.76,100,0,0,176,Buy,USD,
+2011-01-03,GOOG,Stock,,,,15.01,100,0,0,1501,Buy,USD,
+2015-01-01,BTC,CRYPTO,,,,1000.00,2,0,0,2000,Buy,USD,
+2015-01-02,SPY,ETF,,,,178.28,5.6,0,0,1000,Buy,USD,
+2021-10-29,K.TO,Stock,,,,7.93,190,20,0,1526.70,Buy,CAD,
+```
+
+Create a new file with any text editor, copy and paste the above block, and save the file as `~/OpenBBUserData/portfolio/holdings/example.csv`. In Mac OS, the Numbers application has a habit of altering the format of CSV files, be careful to preserve the formatting when saving. A simple text editor works very well for this particular task. If trouble persists while attempting to load a portfolio file, check the file for abnormalities.
+
+To manually set the asset categorization (region, sector, industry, country), use the template file, `Public_Equity_Orderbook.xlsx`, instead.
```

Comment:
Then also remove this comment if you discuss the xlsx instead.

Reply:
Already done, haven't committed the changes yet.